spec/features/job_seekers_can_view_vacancies_spec.rb

```diff
@@ -233,6 +233,38 @@
     end
   end
 
+  describe 'Displaying religious character' do
+    # The linking character links the school type and the religious character
+    # Therefore, if there is no religious character, we expect there to be no linking character
+    let(:linking_character) { ',' }
+
+    context 'when viewing vacancies belonging to a school with a religious character' do
+      scenario 'vacancies display the religious character' do
+        school_with_unique_religion = create(
+          :school,
+          gias_data: { religious_character: 'Unique Religion' })
+        create(:vacancy, school: school_with_unique_religion)
+        Vacancy.__elasticsearch__.client.indices.flush
+        visit jobs_path
+        expect(page).to have_content(linking_character + ' Unique Religion')
+      end
+    end
+
+    context 'when viewing vacancies belonging to a school without a religious character' do
+      scenario 'vacancies do not display the religious character after the school type' do
+        unique_school_type = create(:school_type, label: 'Unique School Type', code: 10)
+        school_with_no_religion = create(
+          :school,
+          school_type: unique_school_type,
+          gias_data: { religious_character: 'None' })
+        create(:vacancy, school: school_with_no_religion)
+        Vacancy.__elasticsearch__.client.indices.flush
+        visit jobs_path
+        expect(page).not_to have_content('Unique School Type' + linking_character)
+      end
+    end
+  end
+
```

Comment:
If you create a presenter method for the school type you can then write unit tests for that method to test the same thing these tests would be testing. It might be better to do that but I'd probably check that with @cesidio @tatyree first 😄

Reply:
Good point, I was forgetting the
> _**testing pyramid**_
which I take to mean that feature tests are expensive.

Reply:
An even cheaper approach would be to write a view test to ensure the business logic in the view does what you're expecting.